packages/deployment/ansible/cosmos-genesis.yml

```diff
@@ -9,8 +9,8 @@
     - BOOTSTRAP_TOKENS: 100provisionpass
     - CHAIN_NAME: "{{ lookup('file', SETUP_HOME + '/ag-chain-cosmos/chain-name.txt') }}"
     - NETWORK_CONFIG_URL: https://testnet.agoric.com/network-config
+    - FAUCET_TOKENS: "{{ lookup('file', SETUP_HOME + '/boot-tokens.txt') }}"
```

Comment:
I'm not confident about reviewing this. Tell me about your testing experience?

Reply:
I spun up a chain from scratch.  Before this change, there was no `urun` in the faucet account.  After this change and spinning up yet another chain from scratch, there was both `urun`, `ubld` (as defined in https://github.com/Agoric/agoric-sdk/blob/da4991d89e06a7282a93afc3bed9821c48e9f86d/packages/deployment/src/setup.js#L5), and `provisionpass`, as expected.